technical/taster-classic-badge-integration.md

```diff
@@ -2,47 +2,39 @@
-| `rate` | Keeps users on your page without directing them back to Taster. For users without JavaScript this behaves like the link panel. With this widget the users can rate and answer survey questions on the page and the contents of the iframe update themselves accordingly |
-
 ## Integration Guide
 
 There are several components and specifications that must be included in your site to allow the slim badge to operate correctly. They should be included on any page that is visible to the user.
 
 1. The head of the document should contain the following script tag:
 
   ```
-  <script type="text/javascript" src="http://www.bbc.co.uk/taster/pilot-lib-slim.js"></script>
+  <script type="text/javascript" src="http://www.bbc.co.uk/taster/pilot-lib.js"></script>
   ```
 
-2. The iframe (which contains the quickrate badge) should be placed on the page with the following lines which initiate an iFrame pointed at the slim badge for your pilot. Please note that `<pilot-id>` should be replaced with the pilot ID that you have been assigned and `<panel_type>` should be relaced with `link` or `rate` according to the desired state described in the table above (if you are uncertain of these settings, please speak to the Taster team or refer to the Testing section at the end of this document):
+2. The iframe (which contains the quickrate badge) should be placed on the page with the following lines which initiate an iFrame pointed at the classic badge for your pilot. Please note that `<pilot-id>` should be replaced with the pilot ID that you have been assigned (if you are uncertain of these settings, please speak to the Taster team or refer to the Testing section at the end of this document):
 
   ```
-  <iframe class="taster-offsite-panel" src="http://www.bbc.co.uk/taster/projects/<pilot-id>/offsite/slim" frameborder="0" scrolling="no"></iframe>
+  <iframe class="taster-offsite-panel" src="http://www.bbc.co.uk/taster/projects/<pilot-id>/offsite/rate" frameborder="0" scrolling="no"></iframe>
   ```
 
 3. At the end of the HTML document, the following code should be entered:
 
   ```
   <script type="text/javascript">
-      require(['pilot-lib-slim/taster-offsite-panel'], function (Panel) {
+      require(['pilot-lib/taster-offsite-panel'], function (Panel) {
           var panel = new Panel('.taster-offsite-panel');
       });
   </script>
   ```
 
-4. In order for the badge to operate correctly, certain CSS rules need to applied to the iframe as follows:
+4. In order for the badge to operate correctly, certain CSS rules need to applied to the iframe. An example has been provided but these may be adjusted within certain limits depending on the requirements of your pilot. Please speak to the Taster team for more information. Typical CSS may be as follows:
 
   ```
   .taster-offsite-panel {
-    position: fixed;
-    right: 0;
+    position: absolute;
```

Comment:
I'd suggest removing `position: absolute` and just have it static in the flow but that's a minor point